sql/src/test/java/io/crate/analyze/BlobTableAnalyzerTest.java

```diff
@@ -204,4 +205,14 @@ public void testCreateBlobTableWithInvalidShardsParam() throws Exception {
         expectedException.expectMessage("invalid number 'foo'");
         e.analyze("create blob table screenshots clustered into ? shards", new Object[]{"foo"});
     }
+
+    @Test(expected = OperationOnInaccessibleRelationException.class)
```

Comment:
We tend to favour using `expectedException.expect` instead. Not a big issue but since it's already used elsewhere in the test it would be nice to keep things consistent.

Reply:
Sure thing, I'll change that.